src/main/java/de/tum/in/www1/artemis/web/rest/RatingResource.java

```diff
@@ -0,0 +1,115 @@
+package de.tum.in.www1.artemis.web.rest;
+
+import static de.tum.in.www1.artemis.web.rest.util.ResponseUtil.forbidden;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import de.tum.in.www1.artemis.domain.Rating;
+import de.tum.in.www1.artemis.domain.Result;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.domain.participation.StudentParticipation;
+import de.tum.in.www1.artemis.service.AuthorizationCheckService;
+import de.tum.in.www1.artemis.service.ParticipationService;
+import de.tum.in.www1.artemis.service.RatingService;
+import de.tum.in.www1.artemis.service.ResultService;
+import de.tum.in.www1.artemis.service.UserService;
+
+/**
+ * REST controller for managing Rating.
+ */
+@RestController
+@RequestMapping("/api")
+public class RatingResource {
+
+    private static final String ENTITY_NAME = "rating";
+
+    private final Logger log = LoggerFactory.getLogger(RatingResource.class);
+
+    private final RatingService ratingService;
+
+    private final UserService userService;
+
+    private final AuthorizationCheckService authCheckService;
+
+    private final ResultService resultService;
+
+    private final ParticipationService participationService;
+
+    public RatingResource(RatingService ratingService, UserService userService, AuthorizationCheckService authCheckService, ResultService resultService,
+            ParticipationService participationService) {
+        this.ratingService = ratingService;
+        this.userService = userService;
+        this.authCheckService = authCheckService;
+        this.resultService = resultService;
+        this.participationService = participationService;
+    }
+
+    /**
+     * Return Rating referencing resultId or null
+     *
+     * @param resultId - Id of result that is referenced with the rating
+     * @return Rating or null
+     */
+    @GetMapping("/results/{resultId}/rating")
+    @PreAuthorize("hasAnyRole('USER', 'TA', 'INSTRUCTOR', 'ADMIN')")
+    public ResponseEntity<Optional<Rating>> getRatingForResult(@PathVariable Long resultId) {
+        Optional<Rating> rating = ratingService.findRatingByResultId(resultId);
```

Comment:
This means, everyone can see every rating in all courses, not a good solution. 
I think we should only allow this to users for results for their own participations.
TAs and instructors can see all ratings in the course in which they are TAs / instructors.
Admins can see all ratings

Reply:
I've discussed this with @jpbernius and for now we will only allow the student who rated the result to see it. 

Your suggested changes would follow at a later point. Is that ok for you?

Reply:
That would be ok. It's just important to have access control here. You can easily integrate admins as well, I think this would be good! Please leave a TODO for tutors and instructors

Reply:
Admins are now allowed as well and a TODO for instructors is added!